src/js/actions/setting-actions.js

```diff
@@ -19,7 +19,7 @@ export default class SettingActions {
   fetch() {
     co(function* fetch() {
       try {
-        const data = yield api.fetch(API_PATH);
+        const data = yield api.fetch(SETTINGS_API_PATH);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'yield' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).